lib/vdsm/storage/blockVolume.py

```diff
@@ -35,7 +35,6 @@
 from vdsm.storage import lvm
```

Comment:
Lets move this to first patch so we can simplify _reloadlvs.

Reply:
Also, can you find the commit that remove the code using BlockVolume.rename() and
add

    Fixes: {commit-hash}

to the commit message?